_clients/java.md

```diff
@@ -33,7 +33,8 @@ If you're using Gradle, add the following dependencies to your project:
 
 ```
 dependencies {
-  implementation 'org.opensearch.client:opensearch-java:2.6.0'
+  implementation 'org.opensearch.client:opensearch-java:2.8.1'
+  implementation 'org.apache.httpcomponents.client5:httpclient:5.2.1'
```

Comment:
Also i added gradle dependency guide for httpclient 5. It seems omitted in https://github.com/opensearch-project/documentation-website/pull/5377